### ISSUE TITLE
Add redirect from mefiscripts.github.io

### ISSUE THREAD

**Issue by jcushman** [mefiscripts/mefiscripts#27]
@emanuelfeld just sent a neat pull request that adds a page for all the scripts at https://mefiscripts.github.io/mefiscripts/ . Awesome!

Currently plain https://mefiscripts.github.io/ without the subdirectory is a 404, though -- the only way to host content there is from a repo called `mefiscripts.github.io`.

@emanuelfeld, in your PR you suggested renaming this repo to `mefiscripts.github.io`, but I'm leaning against that -- it seems like a messy name, based on a github implementation detail that won't be relevant to most people. I did set up a new repo under that name, though, so we can host _something_ there. Any chance you'd be up for putting something together that just redirects to https://mefiscripts.github.io/mefiscripts/ with an HTML redirect? I'm not quite up to speed on Github Pages and Jekyll ...

**Comment by emanuelfeld** [comment]
Added. I think you might need to change the settings to deploy as a gh pages site, though. My concern wasn't so much about contributors to the repo, but about people visiting https://mefiscripts.github.io/mefiscripts/, which looks weirder than https://mefiscripts.github.io.

You could always have mefiscripts.github.io be the main repo and keep the mefiscripts repo with a README that says "go to the other repo."

**Comment by emanuelfeld** [comment]
Occurred to me that if you do decide to use mefiscripts.github.io, you would want to rename this repository and not maintain a repository called mefiscripts. Otherwise GitHub will not do redirects from the old repo and links posted on Metafilter/Userscript update urls will likely break for some people.

https://github.com/blog/1508-repository-redirects-are-here

**Comment by jcushman** [comment]
Thanks! I got this to publish, so https://mefiscripts.github.io now redirects to https://mefiscripts.github.io/mefiscripts/ . I tweaked the code to use the jekyll-redirect-from plugin, which basically does the same thing but includes a clickable link and refresh meta tag as well as JS.

I'm thinking of leaving this repo at `mefiscripts` for now, on the grounds that making the project welcoming to developers seems more pressing than making it welcoming to readers. Definitely open to more conversation though if folks disagree.
